package.xml

```diff
@@ -17,6 +17,7 @@
   <author>Ken Conley</author>
 
   <buildtool_depend version_gte="0.5.74">catkin</buildtool_depend>
+  <run_depend version_gte="0.5.74">catkin</run_depend>
```

Comment:
So the buildtool depend should have a minimum version, but the run depend should not?